packages/@aws-cdk/assert/jest.ts

```diff
@@ -1,4 +1,4 @@
-import { Stack } from '@aws-cdk/core';
+import * as core from '@aws-cdk/core';
```

Comment:
@rix0rrr I changed this because the rewriter couldn't handle these very well and I didn't want to bother right now.

Reply:
That's unfortunate. I was planning to make that work. What was the issue?

Reply:
The current rewriter will produce the following code:

```ts
import { core } from 'monocdk-experiment';
let { Stack } = core;
```

This doesn't compile in TypeScript.

I believe we should produce the following code:

```ts
import { core } from 'monocdk-experiment';
type Stack = core.Stack;
```

Shouldn't be too hard, but I since I want to move all core types to the root, then I thought I would update the rewriter as I do that.

Reply:
> This doesn't compile in TypeScript.

No, it sure does compile. The problem here is probably that the variable assignment uses `let` instead of `const` and the compiler/linter says that the variable is never reassigned.

Problem here was that I couldn't figure out how to make it a `const` VariableDeclaration.

Reply:
I’ll try again but from my experiments “const” did not work as wel

Reply:
I’ll let this merge and will look into the rewriter next.